### PR TITLE
DOC: Add CoC to the README

### PR DESCRIPTION
**Pull request by sparalic**
@datapythonista : Add CoC to README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project you are expected to abide by pandas' code of conduct. More information can be found at: [pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (gfyoung) [inline README.md]:
```suggestion
As contributors and maintainers to this project, you are expected to abide by pandas' code of conduct. More information can be found at: [pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

**Comment by TomAugspurger** [comment]
Sorry about the CI failures again @sparalic. If you merge master and push, things should be fixed

```
$ git fetch upstream
$ git merge upstream/matser
$ # git add and commit stuff
$ git push origin pandas-coc
```

You can also put in @gfyoung's request change at the same time if you want

**Comment by sparalic** [comment]
@TomAugspurger  and @gfyoung thank you for all of your help. @TomAugspurger, I tried merging with the master and pushing as you suggested, but some of the CI test failed again.

**Comment by gfyoung** [comment]
@sparalic : I restarted the failing build for you to see if the failure goes away.  If not, you should merge with `master` and push.

File: README.md [hunk 2]
```diff
@@ -233,3 +233,5 @@ You can also triage issues which may include reproducing bug reports, or asking
 Or maybe through using pandas you have an idea of your own or are looking for something in the documentation and thinking ‘this can be improved’...you can do something about it!
 
 Feel free to ask questions on the [mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata) or on [Gitter](https://gitter.im/pydata/pandas).
+
+As contributors and maintainers to this project, you are expected to abide by pandas' code of conduct. More information can be found at: [pandas Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```
Comment (datapythonista) [inline README.md]:
```suggestion
As contributors and maintainers to this project, you are expected to abide by pandas' code of conduct. More information can be found at: [Contributor Code of Conduct](https://github.com/pandas-dev/pandas/blob/master/.github/CODE_OF_CONDUCT.md)
```

I think you changed this before, but was lost somehow (better if the name of the link matches the title of the target page). Everything is green now, if you can just apply this change (can be done from the GitHub web interface I think), I think this is ready to be merged. Thanks!

**Comment by sparalic** [comment]
@datapythonista and @gfyoung thank you!

**Review by datapythonista — APPROVED** [review]
lgtm, thanks @sparalic

**Comment by TomAugspurger** [comment]
You are just not having luck with our CI @sparalic 😄  The windows failure is unrelated. Tracking it in https://github.com/pandas-dev/pandas/issues/27902.

This is good to merge once the doc build passes.

**Comment by TomAugspurger** [comment]
Thanks @sparalic!

**Pull request merged**